spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/MklInt8Convertible.scala

```diff
@@ -282,7 +283,14 @@ trait MklInt8Convertible {
   private def getWeight(module: AbstractModule[_, _, Float]): Tensor[Float] = {
     if (module != null) {
       // the getParameters will flatten the weight and bias, it's wrong
-      module.parameters()._1(0)
+      val weight = module.parameters()._1(0)
+      // if the weight is came from nn.SpatialConvolution and the nGroup is 1,
+      // we need to skip the first dimension
+      if (module.isInstanceOf[SpatialConvolution[Float]] && weight.size(1) == 1) {
```

Comment:
Hi Yanzhang, could you please explain the reason why we need to skip the first dimension of nn.SpatialConv with nGroup 1? So we could take this into consideration in future development, thanks!

Reply:
Got it.